BackendServices/AuxiliaryServices/WebUtils/WebUtils.csproj

```diff
@@ -5,6 +5,10 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
   </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNet.WebApi.Client" Version="6.0.0" />
+  </ItemGroup>
```

Comment:
Maybe a mistake?

Reply:
This got slipped in by accident for the GriefReport changes that will come but technically.

Reply:
I had to add this because i cannot parse multipart formdata PROPERLY because i have no access to HTTPRequest from within Backend project, thus making it more impossible to parse multipart if it contains MULTIPLE files.

Reply:
![image](https://github.com/GitHubProUser67/MultiServer3/assets/26616362/a72fb0eb-01b2-412f-ad2b-3ba502e95929)
So I came up with this.